miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditBoxController.java

```diff
@@ -77,6 +81,18 @@ public Boolean isScannerEnabled() {
     return boxService.getColumnSizes();
   }
 
+  @InitBinder
+  public void includeForeignKeys(WebDataBinder binder) {
+    binder.registerCustomEditor(StorageLocation.class, new PropertyEditorSupport() {
+      @Override
+      public void setAsText(String text) throws IllegalArgumentException {
+        StorageLocation location = new StorageLocation();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)